Source/items.cpp

```diff
@@ -496,7 +496,8 @@ void CalcSelfItems(Player &player)
 
 			if (currstr >= equipment._iMinStr
 			    && currmag >= equipment._iMinMag
-			    && currdex >= equipment._iMinDex)
+			    && currdex >= equipment._iMinDex
+			    && ((equipment._iClass == ICLASS_MISC || equipment._iClass == ICLASS_GOLD || equipment._iClass == ICLASS_QUEST) || equipment._iDurability != 0))

```

Comment:
This could use the `IsAnyOf` template like the check in `Player::CanUseItem`. I reckon it would be worth moving this to a method of the Item class that takes the stats to use in the comparison as arguments now that the check is more complicated.

Something like (using unsigned given stats should be non-negative, int is probably more appropriate in the current codebase 😂 ):
```cpp
Item::IsUsable(unsigned strength, unsigned magic, unsigned dexterity) const
{
	return strength >= _iMinStr
		    && magic >= _iMinMag
		    && dexterity >= _iMinDex
		    && (IsAnyOf(_iClass, ICLASS_MISC, ICLASS_GOLD, ICLASS_QUEST) || _iDurability != 0);
}
```

Lets you simplify this use to `if(equipment.IsUsable(currstr, currmag, currdex))` and Player::CanUseItem to
```cpp
Player::CanUseItem(const Item &item) const
{
	return item.IsUsable(_pStrength, _pMagic, _pDexterity);
}

Reply:
May I suggest extracting the last section further into a `Item::IsBroken` method? Then the original would read:
```cpp
Item::IsUsable(unsigned strength, unsigned magic, unsigned dexterity) const
{
	return strength >= _iMinStr
		    && magic >= _iMinMag
		    && dexterity >= _iMinDex
		    && !IsBroken();
}
```